compiler/rustc_codegen_llvm/src/debuginfo/mod.rs

```diff
@@ -56,44 +57,39 @@ const DW_TAG_auto_variable: c_uint = 0x100;
 const DW_TAG_arg_variable: c_uint = 0x101;
 
 /// A context object for maintaining all state needed by the debuginfo module.
-pub struct CrateDebugContext<'a, 'tcx> {
+pub struct CodegenUnitDebugContext<'a, 'tcx> {
```

Comment:
While you're changing things, can you use `'ll` for the first lifetime here, as well as in the inherent impl below? I've made that change in my own PR that touches this struct, but it would likely better fit here.

Reply:
Yes, that makes sense.